x-pack/plugin/data-frame/src/main/java/org/elasticsearch/xpack/dataframe/persistence/DataFrameInternalIndex.java

```diff
@@ -194,6 +195,9 @@ private static XContentBuilder addDataFrameTransformsConfigMappings(XContentBuil
                         .field(TYPE, KEYWORD)
                     .endObject()
                 .endObject()
+            .endObject()
+            .startObject(DataFrameTransformConfig.DESCRIPTION.getPreferredName())
+                .field(TYPE, TEXT)
```

Comment:
Text fields are analysed is this what you want

Reply:
@davidkyle, I thought it would be nice to be able to search for dataframe configs via their descriptions. What do you think @hendrikmuhs ?

Reply:
sounds reasonable to me to provide full text search for it.

Devil's advocate: Let's limit the size of the description.

Reply:
Will limit it to 1k